src/dependencies.jl

```diff
@@ -655,10 +870,10 @@ function can_provide(p::PackageManager,opts,dep)
     end
 end
 
-issatisfied(dep::LibraryDependency) = !isempty(_find_library(dep))
+issatisfied(dep::Dependency) = !isempty(_find_dependency(dep))
 
-allf(deps) = [dep => _find_library(dep) for dep in deps.deps]
-function satisfied_providers(deps::LibraryGroup, allfl = allf(deps))
+allf(deps) = [dep => _find_dependency(dep) for dep in deps.deps]
```

Comment:
This is deprecated syntax but IIRC there's no `@compat` for it. I think you can do
```julia
Dict([Pair(dep, _find_dependency(dep)) for dep in deps.deps])
```
to get around that?